components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -7,21 +7,66 @@ import {Channel} from 'mattermost-redux/types/channels';
 
 import Constants from 'utils/constants';
 
+import ChannelMentionBadge from './channel_mention_badge';
 import SidebarBaseChannel from './sidebar_base_channel';
 import SidebarDirectChannel from './sidebar_direct_channel';
 import SidebarGroupChannel from './sidebar_group_channel';
 
 type Props = {
+
+    /**
+     * The channel object for this channel list item
+     */
```

Comment:
0/5, but these comments may not be necessary for props that are more obvious. We went through a phase of adding them to everything in the web app, but they weren't always helpful.

Reply:
I like to be verbose, and consistency also looks better. But yeah I can see where some are overkill.